### ISSUE TITLE
Impement rax2 -s and rax2 -S in r2 (maybe ?s and ?S) always append newline

### ISSUE THREAD

**Issue by radare** [radare/radare2#5877]

**Comment by Maijin** [comment]
![image](https://cloud.githubusercontent.com/assets/1408600/18934486/4177766a-85da-11e6-8d68-7dd180f78fea.png)

**Comment by Maijin** [comment]
![image](https://cloud.githubusercontent.com/assets/1408600/18934494/576a1afe-85da-11e6-958e-ea5d267b3908.png)

**Comment by Greynad** [comment]
![r2-mod](https://cloud.githubusercontent.com/assets/5173529/18934979/49973dbe-85dd-11e6-99d3-cf45be98c3d0.jpg)

@Maijin Would that be okay? (help style wise)

**Comment by Maijin** [comment]
Why not ?s and ?S like suggested by @radare

**Comment by radare** [comment]
well. that's inconsistent

<img width="354" alt="screen shot 2016-09-29 at 10 40 51" src="https://cloud.githubusercontent.com/assets/917142/18946926/50f1c2ec-8631-11e6-8766-c997b35107b0.png">

**Comment by Greynad** [comment]
Why not ?s or  ?S: because those names are already used with
?s from to step  sequence of numbers from to by steps
?S addr              return section name of given address

Plus as said yesterday, the function is already implemented and available in your current build, it’s just badly documented.

![r2_mod](https://cloud.githubusercontent.com/assets/5173529/18959952/3e875b8a-8668-11e6-9080-0aacdbafd016.jpg)


All I did was update the doc string to make it a bit clearer (not well enough given the miscomprehension)
Now if you feel like using an entirely different syntax to convert to ascii is appropriate, I suggested ?a which is available.
Tell me which you like best.

![r2_mod2](https://cloud.githubusercontent.com/assets/5173529/18959972/49d40556-8668-11e6-95ed-a57258ac8a6c.jpg)
